share/locale/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ install(DIRECTORY ./
     DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}locale
     FILES_MATCHING
     REGEX ".*\\.qm"
+    REGEX ".*_en\\.qm" EXCLUDE
```

Comment:
Do not install the 'source' language, those should get taken from the compiled in strings. Before this PR this only (potentially) was an issue for the tours' translations though.